contrib/kind.sh

```diff
@@ -544,6 +552,18 @@ kubectl_wait_pods() {
   fi
 }
 
+docker_create_second_interface() {
+  echo "adding second interfaces to nodes"
+
+  # Create the network as dual stack, regardless of the type of the deployment. Ignore if already exists.
+  docker network create --ipv6 --driver=bridge network2 --subnet=172.19.0.0/16 --subnet=fc00:f853:ccd:e798::/64 || true
```

Comment:
should this network be called kind2 or something?

Reply:
I wanted to make it clear it's not related to the primary network, thought that something different from kind* was better.
Can change to "exgwnetwork" or something along that line though